prac_05/wimbledon.py

```diff
@@ -0,0 +1,49 @@
+"""
+CP1404 Practical 5
+Wimbledon
+"""
+
+FILENAME = "wimbledon.csv"
+COUNTRY_INDEX = 1
+CHAMPION_INDEX = 2
+
+
+def main():
+    records = get_past_records(FILENAME)
```

Comment:
I forgot about this in my code too, but using 'get' in a function name can imply you're getting input from the user. Since our data is from a file, 'load' is a pretty good term